config/formats.js

```diff
@@ -797,7 +797,649 @@ exports.Formats = [
 
 	// Randomized Metas
 	///////////////////////////////////////////////////////////////////
+	{
+		name: "Wisp Super Staff Bros.",
+		section: "Other Metagames",
+		mod: 'wssb',
+		team: 'randomWispStaffBros',
+		ruleset: ['Pokemon', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
+		onBegin: function () {
+			// This seasonal gets a bit from Super Smash Bros., that's where the initial message comes from.
+			this.add('message', "3");
+			this.add('message', "2");
+			this.add('message', "1");
+			this.add('message', "GET READY!");
+			this.add('message', "GOOOO!!!");
+			this.add('raw|<div class="infobox"><center><b>Wisp Super Staff Bros Credits:</b></center><b>%Emg TechP Volco</b> - Programming, Organization, Testing<br /><b>@LotusChamp Torpid</b> - Programming, Organization, Testing<br /><b>Wisp Staff Members</b> - Cooperation and understanding about the project (and some provided aid)');
```

Comment:
why?
... no...?

comment credits should be enough, and the credits command is even pushing it...